spec/features/course/achievement_management_spec.rb

```diff
@@ -108,6 +108,158 @@
           expect(achievement.reload.description).to eq(new_description)
         end
       end
+
+      context 'achievement achievement conditions' do
+        let(:achievement) { create(:course_achievement, course: course) }
+        it { is_expected.to have_selector(:link_or_button,
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.